cmd/statusd/main.go

```diff
@@ -92,7 +93,7 @@ var (
 // All general log messages in this package should be routed through this logger.
 var logger = log.New("package", "status-go/cmd/statusd")
 
-func main() {
+func init() {
```

Comment:
I would keep flag parsing in `main()`. Imagine, `init` function will be moved into another file or just to the bottom of file, and anyone looking at the `func main() {` will loose the fact that the flag parsing happens.
Also, let's keep PRs as small as possible – if there is a sense to refactor main/init code, let's do it in a separate PR, it has nothing to do with fleet configuration.

Reply:
I am not sure why `init()` would be moved anywhere else. It is executed before `main()` so logically it should be before it.

I needed to make some changes because the logger in `NodeConfig` was improperly initialized before we initialize the root logger object.

Reply:
Hmm, okay, if there is a valid reason than it's fine. Generally, init should be avoided unless really needed, because init() is one of those rare "magical" things that Go has, and they can easily fool/conceal  important aspects of the program. Anyway,